source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -9,25 +9,28 @@ review_in: 6 months
 
 You must have authenticated your users before you can prove their identity.
 
-If you [requested identity proving][integrate.choose-level-of-confidence], when you [retrieve user information with `/userinfo`][integrate.retrieve-user-info], you’ll receive a response containing additional claims (user attributes). You may receive different claims, depending on how your user proved their identity.
+If you [requested identity proving][integrate.choose-level-of-confidence], when
+you [retrieve user information with `/userinfo`][integrate.retrieve-user-info], you’ll receive a response containing
+additional claims (user attributes). You may receive different claims, depending on how your user proved their identity.
 
-Your service’s needs will determine how you process the other claims that GOV.UK One Login provides about your user. You’ll probably need to match against information held by your service or organisations you work with.
+Your service’s needs will determine how you process the other claims that GOV.UK One Login provides about your user.
+You’ll probably need to match against information held by your service or organisations you work with.
```

Comment:
Just wondering why we need the word probably here, in what case would you not need to match against info held by your dervice/org - does it depend on your service needs basically? We just like to avoid being vague if we can so thought I would check if we could snip it

Reply:
I didn't write this, my editor just inserted a new line (which won't be reflected in the real site)

Reply:
Okay thanks